terraform/workspace_variables/preprod.backend.tfvars

```diff
@@ -0,0 +1,3 @@
+storage_account_name = "s165t01preprodtfstate"
```

Comment:
This will not be stored in here. We will create a seperate tf state storage and KV for qts service. so tfvars files for test. preprod and prod should not be in this PR.